cmd/kubelet/app/options/options.go

```diff
@@ -76,6 +76,12 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.TLSPrivateKeyFile, "tls-private-key-file", s.TLSPrivateKeyFile, "File containing x509 private key matching --tls-cert-file.")
 	fs.StringVar(&s.CertDirectory, "cert-dir", s.CertDirectory, "The directory where the TLS certs are located (by default /var/run/kubernetes). "+
 		"If --tls-cert-file and --tls-private-key-file are provided, this flag will be ignored.")
+	fs.StringVar(&s.BootstrapAuthToken, "bootstrap-auth-token", s.BootstrapAuthToken, ""+
```

Comment:
@liggitt I understand. My thought is that besides getting tls certs, the future goal of this work and other related bootstrap work, is to enable users **not** to provide kubconfig when bootstrapping the cluster.

But given that other work are not ready yet, so I am open to use what's in the kubeconfig to create the rest client, and maybe we can make changes later to have a flag (e.g. `--cluster` in the proposal I pasted above) to override the kubeconfig for this bootstrap auth purpose.

Are you guys on the same page with this? @mikedanese @gtank @aaronlevy